apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/event/api/EventController.java

```diff
@@ -0,0 +1,35 @@
+package edu.pw.apsienrollment.event.api;
+
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import lombok.RequiredArgsConstructor;
+
+import edu.pw.apsienrollment.event.EventService;
+import edu.pw.apsienrollment.event.db.Event;
+
+@RestController
+@RequestMapping("events")
+@RequiredArgsConstructor
+public class EventController {
+    private final EventService eventService;
+
+    @GetMapping("")
```

Comment:
Please, add Swagger annotations to enable manual testing of API

Reply:
Done.